extensions/wikia/AdEngine/js/PageFairDetection.js

```diff
@@ -0,0 +1,71 @@
+define('ext.wikia.adEngine.pageFairDetection', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.utils.scriptLoader',
+	'wikia.browserDetect',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (adContext, scriptLoader, browserDetect, doc, log, win) {
+	'use strict';
+
+	var context = null,
+		logGroup = 'ext.wikia.adEngine.pageFairDetection';
+
+	function getContext() {
```

Comment:
Why not `var context = adContext.getContext();` and remove this method entirely?

I understand that it may be called to early on Mercury then just explicitly call `adContext.getContext();` everywhere.